xml/hwbp_registers.xml

```diff
@@ -746,6 +749,12 @@
             \Rxepc or \RcsrDpc (depending on \FcsrMcontrolSixAction) must be set
             to the virtual address of the instruction that matched.
 
+            If an instruction partially executed, the reported timing is still
+            before. This might mean that when the instruction is executed again
+            it will perform some of the same stores again (e.g. \tt{cbo.zero}),
+            or the instruction might have a mechanism that avoids duplicating the
+            stores (e.g. vector stores).
```

Comment:
The "vector store" reference leaves the crucial piece of information a mystery to the reader unless they are intimately familiar with RVV.  The parenthetical should say:

(e.g. vector stores and the vstart mechanism).

Reply:
Done. Thanks.